specification/workloadmonitor/resource-manager/Microsoft.WorkloadMonitor/preview/2020-01-13-preview/Microsoft.WorkloadMonitor.json

```diff
@@ -0,0 +1,805 @@
+      "get": {
+        "summary": "Get history of a monitor of a resource (with optional filter).",
+        "operationId": "MonitorStateChanges_List",
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/resourceNamespace"
+          },
+          {
+            "$ref": "#/parameters/resourceType"
+          },
+          {
+            "$ref": "#/parameters/resourceName"
+          },
+          {
+            "$ref": "#/parameters/monitorId"
+          },
+          {
+            "$ref": "#/parameters/filter"
+          },
+          {
+            "$ref": "#/parameters/expand"
+          },
+          {
+            "in": "query",
+            "name": "startTimestampUtc",
+            "description": "The start Timestamp for the desired history",
+            "required": false,
+            "type": "string",
+            "format": "date-time"
+          },
+          {
+            "in": "query",
+            "name": "endTimestampUtc",
+            "description": "The end Timestamp for the desired history",
+            "required": false,
+            "type": "string",
+            "format": "date-time"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "search results matching criteria",
+            "schema": {
+              "$ref": "#/definitions/MonitorStateChangeList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/DefaultError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get monitor history with default values": {
+            "$ref": "./examples/MonitorHistory_GetDefault.json"
+          },
+          "Get monitor list with filtered values": {
+            "$ref": "./examples/MonitorHistory_GetFiltered.json"
+          },
+          "Get monitor history with expanded values": {
+            "$ref": "./examples/MonitorHistory_GetExpanded.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{resourceNamespace}/{resourceType}/{resourceName}/providers/Microsoft.WorkloadMonitor/monitors/{monitorId}/history/{timestampUnix}": {
+      "get": {
+        "summary": "Get the status of a monitor at a specific timestamp in history.",
+        "operationId": "MonitorHistoricalStateChange_Get",
```

Comment:
Likely this should be 'Monitors_GetStateChange'

Reply:
Changed to MonitorHistory_GetStateChanges